### ISSUE TITLE
Invalid expression term 'void' CS1525

### ISSUE THREAD

**Issue by goryaynov-leonid** [mamift/LinqToXsdCore#70]
Hello, I use LinqToXsd CLI tool to generate CS models from XSD schemas, and when I'm trying to build CS model generated from attached XSD sxhema got a compilation error CS1525 in lines where used `void.TypeDefinition` (e. g. in lines 58, 84 etc.). 

Atteach: XSD schema, XSD config for LinqToXsd that I used and generated CS file (for all of them add `.txt` extension to upload on github)

Is that a bug? Any suggestions how to fix that?

[cda.xsd.config.txt](https://github.com/user-attachments/files/18057050/cda.xsd.config.txt)
[STRUCTURED_BODY.xsd.txt](https://github.com/user-attachments/files/18057051/STRUCTURED_BODY.xsd.txt)
[STRUCTURED_BODY.XSD.cs.txt](https://github.com/user-attachments/files/18057052/STRUCTURED_BODY.XSD.cs.txt)

**Comment by mamift** [comment]
Yes this is a bug, there are certain features of XSD that the tool does not yet map properly. Are you able to provide a complete reproducible example of the XSD files? STRUCTURED_BODY.xsd references more XSD files like:
```
<xsd:import schemaLocation="fias.xsd" namespace="urn:hl7-ru:fias" />
  <xsd:import schemaLocation="address.xsd" namespace="urn:hl7-ru:address" />
  <xsd:import schemaLocation="identity.xsd" namespace="urn:hl7-ru:identity" />
  <xsd:include schemaLocation="coreschemas/datatypes.xsd" />
  <xsd:include schemaLocation="coreschemas/voc.xsd" />
  <xsd:include schemaLocation="coreschemas/NarrativeBlock.xsd" />
```
